import_processor/readme/USAGE.rst

```diff
@@ -0,0 +1,31 @@
+This module provides a set of pre-defined functions and wizards to facilitate importing data into Odoo.
+
+The module offers predefined variables that can be used when writing code snippets in the view.
+
+Note: Only pre-defined variables can be used in code snippets.
+
+The module consists of three main processes:
+
+1. Pre-Processor:
+   - At this stage, you can define code snippets that will be executed before importing the records in the view.
+
+2. Processor:
+   - At this stage, you can define code snippets that will be executed during the import process.
```

Comment:
`that will be executed on each entry or chunk of entries during the import process`